openwisp_radius/settings.py

```diff
@@ -140,6 +140,16 @@ def get_settings_value(option, default):
     raise ImproperlyConfigured('Failed to import PRIVATE_STORAGE_INSTANCE')
 
 CALLED_STATION_IDS = get_settings_value('CALLED_STATION_IDS', {})
+
+for organization in CALLED_STATION_IDS.keys():  # pragma: no cover
+    try:
+        UUID(organization)
+    except ValueError:
+        logger.warning(
+            'Organization slug in CALLED_STATION_IDS setting is deprecated.'
+            ' It will be removed in future, please replace'
+            f' organization slug: {organization} with its id.'
```

Comment:
Minor thing: can you change this string so that the initial spaces are moved to the last character of the preceding line please?